docs/reference/kernel-upload-rights.rst

```diff
@@ -0,0 +1,116 @@
+Kernel upload rights
+####################
+
+Those who are allowed to upload the kernel to the Ubuntu archive have a serious
+responsibility.
+
+To obtain per package upload rights for the Ubuntu kernel, you can apply to
```

Comment:
```suggestion
To obtain per package upload rights for the Ubuntu kernel, you must apply to
```

"can" sounds like a suggestion. unless there is another way to obtain upload rights?